virttest/utils_hotplug.py

```diff
@@ -62,6 +63,8 @@ def create_mem_xml(tg_size, pg_size=None, mem_addr=None, tg_sizeunit="KiB",
         src_xml.pagesize_unit = pg_unit
         src_xml.nodemask = node_mask
         mem_xml.source = src_xml
+    if mem_discard:
```

Comment:
since mem_discard="no" as default, so by default  mem_xml.mem_discard will have value with mem_discard, so it will impact current all cases ,which use create_mem_xml . Are you sure it is your intentional?

Reply:
The lines 66- 67 means:
If mem_discard is defined, the domain xml will include it by using the value given. If it's not defined , the domain xml will not include it.  Any problem?

Reply:
mem_discard is one string ,not boolean , if mem_discard is always True